gnomad/resources/grch37/gnomad.py

```diff
@@ -156,6 +156,22 @@ def _public_browser_gene_ht_path() -> str:
     return "gs://gnomad-public-requester-pays/resources/grch37/browser/gnomad.genes.GRCh37.GENCODEv19.pext.ht"
 
 
+def _public_mnv_path(distance: str = "1") -> str:
```

Comment:
here and in below function i think it would make sense to make distance an int rather than a string